hkl/tests/test_diffract.py

```diff
@@ -165,10 +165,16 @@ def test_forward_solutions_table(fourc):
     # (100) has chi ~ 0 which poses occasional roundoff errors
     # (sometimes -0.00000, sometimes 0.00000)
     sol = fourc.forward(1, 0, 0)
-    assert pytest.approx(sol.omega, 1e-5) == -30
-    assert pytest.approx(sol.chi, 1e-5) == 0
-    assert pytest.approx(sol.phi, 1e-5) == -90
-    assert pytest.approx(sol.tth, 1e-5) == -60
+
+    def check_limits_and_value(axis, value, tol=1e-5):
+        assert min(fourc.calc[axis].limits) <= value
+        assert max(fourc.calc[axis].limits) >= value
+        assert pytest.approx(getattr(sol, axis), 1e-5) == value
```

Comment:
```suggestion
        assert pytest.approx(getattr(sol, axis), rel=tol) == value
```

I believe this was intended to use the parameter, `tol` of the enclosing function.

I think it is worth explicitly [naming the tolerance parameter](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-approx) to `pytest.approx()` which accepts both `rel`ative and `abs`olute tolerances.

Reply:
Good catch.  In this context, `abs` is the preference.  5 digits, 10 microdegrees.  `rel` would evaluate to more digits for angles close to zero.

Reply:
Also applied to four other instances in this file.